src/azure-cli-core/azure/cli/core/auth/binary_cache.py

```diff
@@ -30,6 +31,10 @@ def __init__(self, file_name):
     @retry()
     def _load(self):
         """Load cache with retry. If it still fails at last, raise the original exception as-is."""
+        # SPython use windows-http and its response is not picklable, so we also don't load it.
+        if is_spython():
+            return {}
```

Comment:
A better solution is to simply disable `http_cache` by setting it to `None`: https://github.com/Azure/azure-cli/blob/8097211221ca09f2da1271ad7b65a1f4d24e187c/src/azure-cli-core/azure/cli/core/auth/identity.py#L98